src/test/java/org/opentripplanner/middleware/persistence/PersistenceUtil.java

```diff
@@ -92,14 +104,26 @@ public static MonitoredTrip createMonitoredTrip(String userId) {
         monitoredTrip.tripName = "Commute to work";
         monitoredTrip.tripTime = "07:30";
         monitoredTrip.leadTimeInMinutes = 30;
-        monitoredTrip.monday = true;
-        monitoredTrip.tuesday = true;
-        monitoredTrip.wednesday = true;
-        monitoredTrip.thursday = true;
-        monitoredTrip.friday = true;
+        monitoredTrip.updateWeekdays(true);
         monitoredTrip.excludeFederalHolidays = true;
         monitoredTrip.queryParams = "fromPlace=28.54894%2C%20-81.38971%3A%3A28.548944048426772%2C-81.38970606029034&toPlace=28.53989%2C%20-81.37728%3A%3A28.539893820446867%2C-81.37727737426759&date=2020-05-05&time=12%3A04&arriveBy=false&mode=WALK%2CBUS%2CRAIL&showIntermediateStops=true&maxWalkDistance=1207&optimize=QUICK&walkSpeed=1.34&ignoreRealtimeUpdates=true&companies=";
 
+        monitoredTrip.itinerary = createItinerary();
+
+        Persistence.monitoredTrips.create(monitoredTrip);
+        return monitoredTrip;
+    }
+
+    public static MonitoredTrip createMonitoredTrip(String userId, OtpDispatcherResponse otpDispatcherResponse) {
```

Comment:
This may need to be moved to `src/main` (make `tripName` a parameter)

Reply:
In the interest of moving things forward, I'm avoiding this refactor at this time.